impl/src/main/java/io/kroxylicious/testing/kafka/internal/AnonymousAdminSource.java

```diff
@@ -0,0 +1,13 @@
+/*
+ * Copyright Kroxylicious Authors.
+ *
+ * Licensed under the Apache Software License version 2.0, available at http://www.apache.org/licenses/LICENSE-2.0
+ */
+
+package io.kroxylicious.testing.kafka.internal;
+
+import org.apache.kafka.clients.admin.Admin;
+
+public interface AnonymousAdminSource {
```

Comment:
We can always refactor later but why is "Anonymous" part of the API? I see that as, an important, implementation detail. 

To expand on that we could supply a an admin client with fixed credentials which we have also injected into the cluster and satisfy the contract we need from this interface.

Reply:
Good point, it is an implementation detail.